src/components/pages/rules/rulesGrid/rulesGrid.js

```diff
@@ -30,21 +32,26 @@ export class RulesGrid extends Component {
       rulesColumnDefs.lastTrigger
     ];
 
-    // TODO: This is a temporary example implementation. Remove with a better version
     this.contextBtns = {
-      disable: <Btn key="disable">Disable</Btn>,
+      disable: <Btn key="disable" svg={svgs.disableToggle} onClick={this.openStatusFlyout}>{props.t('rules.flyouts.disable')}</Btn>,
```

Comment:
One thing I noticed recently is instead of using the prop.t here use the react18nnext component `Trans` like so

`import { Trans } from 'react-i18next';`

`<Trans i18nKey="rules.flyouts.disable">Disable</Trans>`

If the language changes with the context buttons in use the buttons won't update in the current approve. (this needs to be done for the all the other context buttons too)